src/components/ui/hook-form/index.ts

```diff
@@ -1,5 +1,6 @@
 export { FormDatePicker } from './datepicker';
 export { FormInput } from './input';
+export { InputMoney } from './inputMoney';
```

Comment:
Let's call it Input Price, and also since we have different inputs it will be better to have all input in Input folder and create Factory, like we decided to do with Icons